src/stores/context.store.ts

```diff
@@ -118,6 +129,19 @@ class AppContext {
         localStorage.setItem(this.localStorageName, JSON.stringify(this.state));
     }
 
+    public setUser(user: User) {
```

Comment:
Synes det havde hjulpet på overblikket i denne fil, hvis user-metoder og dataset-metoderne lå samlet. Man skal scrolle meget for at få et overblik

Reply:
Check